Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:bionic
+
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+
+EXPOSE 5005:5005
```

Comment:
I'm not sure this has any effect? I don't really know for sure, but I think the `-p 5005:5005` is doing all the work.

Reply:
removed